src/main/java/racingcar/application/RandomAccelerator.java

```diff
@@ -0,0 +1,13 @@
+package racingcar.application;
+
+import java.util.Random;
+
+public class RandomAccelerator implements Accelerator {
+
+  private final Random random = new Random();
```

Comment:
정적으로 초기화 되도 괜찮을 것 같습니다 🙇 
```suggestion
  private static final Random RANDOM = new Random();
```